doc/setup/deploy.rst

```diff
@@ -115,6 +115,13 @@ in ``/etc/postgresql/9.5/main/postgresql.conf`` set:
   listen_addresses = 'localhost,192.168.1.2'
   port = 5432
   max_connections = 80
+  
+Don't forget to uncomment the # before the ``listen_addresses`` variable or otherwise it will not be checked and used.
```

Comment:
This is the way most software configuration files works, lines starting with a `#` are commented.

Reply:
@mart-e yes I know and for us this is obvious. For new people, who generally read documentation, this might not be so obvious though. I think clarity is never a bad thing, right?

Reply:
Yes and no. Having too much notes may actually being counter productive. One of the reason of 8670f62e3c1d57371e34836b84a0f5079b2ad0e was that there were too many notes and warning on the documentation, making it hard to read and looks like an harder process that it actually is.
We are assuming (probably too optimistically) people have a bit of knowledge about sysadmin and it is not the point to have a tutorial with copy/paste but to give the configuration required for people knowing what they are looking for.

Reply:
@mart-e okay. However, it is just a sentence. It has no warning or info class or anything else around it. Anyways, if this is the blocking part for this PR and you want me to remove it I'll do so.

Reply:
@mart-e friendly reminder. 😉